docs/_docs/snapshots/snapshots.adoc

```diff
@@ -146,9 +146,9 @@ See the link:tools/control-script#checking-snapshot-consistency[Control Script]
 commands.
 
 == Restoring From Snapshot
-
-Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes.
+=== Manual snapshot restore procedure
+In a nutshell, you need to stop the cluster, replace persistence data and other files with the data from the snapshot,
+and restart the nodes.
 
 The detailed procedure looks as follows:
 
```

Comment:
Do the following on each node. Clean the
link:link:persistence/native-persistence#configuring-persistent-storage-directory[`db/{node_id}`] directory separately if
it's not located under the Ignite `work` dir:

Remove the extra lines

Remove the files and sub-directories related to the `{nodeId}` under your `$IGNITE_HOME/work/db` directory. Clean the

Clean what?

The table
below explains what options are supported:

Delete an extra line. The sentence will then look like this: "The table below explains what options are supported:"

````
[cols="1,1",opts="header"]
|===
|Condition | Description
 @@ -178,6 +178,51 @@
 |N > M | Unsupported.
 |===
````